app/validations/recommendation.js

```diff
@@ -0,0 +1,273 @@
+import {
+  validateNumber,
+  validatePresence,
+} from 'ember-changeset-validations/validators';
+import validatePresenceUnlessValue from '../validators/presence-unless-value';
+
+const commentPresenceMessage = 'Please enter a comment';
+const dateMessage = 'Please enter a valid date';
+const numberMessage = 'Please enter a valid non-negative number';
+const recommendationPresenceMessage = 'Please select a recommendation';
+const votelocationPresenceMessage = 'Please enter a vote location';
+
+export const dispositionForAllActionsValidations = {
```

Comment:
Let's go ahead and document this new kind of export (that it's specific to ember-changeset-validations, etc)..